MFVNeutralino/plugins/TriggerFloats.cc

```diff
@@ -79,6 +83,15 @@ void MFVTriggerFloats::produce(edm::Event& event, const edm::EventSetup& setup)
     if (prints) std::cout << "TriggerFloats run " << event.id().run() << " lumi " << event.luminosityBlock() << " event " << event.id().event() << "\n";
     //std::cout << "Starting TriggerFloats.cc" << std::endl;
 
+    std::vector<std::string> metfilternames = {"Flag_goodVertices", "Flag_globalSuperTightHalo2016Filter", "Flag_HBHENoiseFilter", "Flag_HBHENoiseIsoFilter", "Flag_EcalDeadCellTriggerPrimitiveFilter", "Flag_BadPFMuonFilter", "Flag_eeBadScFilter", "Flag_ecalBadCalibFilter"};
+    if ((year==20161) || (year==20162) ){
+      metfilternames = {"Flag_goodVertices", "Flag_globalSuperTightHalo2016Filter", "Flag_HBHENoiseFilter", "Flag_HBHENoiseIsoFilter", "Flag_EcalDeadCellTriggerPrimitiveFilter", "Flag_BadPFMuonFilter", "Flag_eeBadScFilter"};
+    }
```

Comment:
We should double check this against the MET group's Twiki, to be sure we're applying the recommended ones!